lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -160,7 +160,12 @@ def resolve_all(self, requirements, **kwds):
         dependencies = []
 
         is_installed = self.conda_context.has_env(env)
-        if not is_installed and (self.auto_install or kwds.get('install', False)):
+        install = kwds.get('install', None)
+        if install is None:
```

Comment:
@blankenberg could you add a small comment like:
`# default behavior, install dependencies if conda_auto_install is active`?